### ISSUE TITLE
Running the app a second time deletes the previous download

### ISSUE THREAD

**Issue by PeterHartshorn** [timsutton/brigadier#34]
So, after successfully downloading the installESD.pkg, and then having the install failed due to the hard coded 7-zip url now being invalid (manual install of 7-zip worked around that). Then discovering that the download is deleted when the app exits, I ran the app again using -k -I options. The install failed because a log was not found. Good news is that the files were still there, so I ran the app again with admin rights and the first thing it did was delete the files.

What is the point of having a flag (-k) to say keep the files, if their going to be deleted on a subsequent execution? Perhaps a better solution would be to check if the file is there and then attempt to use it.

**Comment by utdrmac** [comment]
Absolutely, 100% upvote! Why does this tool delete any previous downloads? This .pkg is 650MB! It takes a very long time on slow connections (I waited 2 hours last night). If the package has already been downloaded, why delete it!? Makes no sense.
